src/guide/jbs-jdk-bug-system.md

```diff
@@ -206,10 +206,10 @@ Now that the issue is in the right component and has the basic information, the
 1. Ensure the priority is correct.
    * An approach that has been used for getting a consistent view of priority is to consider three aspects of the issue: **Impact** of the issue; **Likelihood** of it occurring; and, whether there is a **Workaround**. The higher the impact and likelihood the higher the priority; then, having a workaround reduces that priority - but mostly where the impact and likelihood aren't that severe.
 1. Ensure the [Affects Version/s]{.jbs-field} field is correct (within reason).
-  * This may involve reproducing the bug, if doing so is fast and easy.
-  * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
-    * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
-  * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
+    * This may involve reproducing the bug, if doing so is fast and easy.
+    * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
+      * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
+    * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
```

Comment:
```suggestion
    * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family and should not go into a future mainline release.
```
You agreed there should be no comma here.

Reply:
actually I think that comma is fine.  I also added another after the main point of the sentence